src/client/components/pages/entities/identifiers.js

```diff
@@ -20,6 +20,49 @@ import PropTypes from 'prop-types';
 import React from 'react';
 
 
+function IdentifierLink({type, value}) {
+	let link;
+	if (type === 'ISBN-10') {
```

Comment:
I'd like to see two main changes to this:

1. Use a switch statement instead of if...elses
2. Use type.id (or typeId) instead of type.label. The id is much less likely to change than the label.
For code legibility, add a comment next to the line with the label, like this (pseudo code):

```
switch(type):
[…]
  case 18: // Wikidata ID
    link = `https://www.wikidata.org/wiki/${value}`;
    break;
[…]
```